pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -200,41 +214,42 @@ func (pl *PodTopologySpread) NormalizeScore(ctx context.Context, cycleState *fra
 		return nil
 	}
 
-	// Calculate the summed <total> score and <minScore>.
+	// Calculate <minScore> and <maxScore>
 	var minScore int64 = math.MaxInt64
-	var total int64
+	var maxScore int64
 	for _, score := range scores {
-		// it's mandatory to check if <score.Name> is present in m.NodeNameSet
-		if _, ok := s.NodeNameSet[score.Name]; !ok {
+		// it's mandatory to check if <score.Name> is present in m.IgnoredNodes
+		if s.IgnoredNodes.Has(score.Name) {
 			continue
 		}
-		total += score.Score
 		if score.Score < minScore {
 			minScore = score.Score
 		}
+		if score.Score > maxScore {
+			maxScore = score.Score
+		}
 	}
 
-	maxMinDiff := total - minScore
+	minMaxSum := minScore + maxScore
 	for i := range scores {
 		nodeInfo, err := pl.sharedLister.NodeInfos().Get(scores[i].Name)
 		if err != nil {
 			return framework.NewStatus(framework.Error, err.Error())
 		}
 		node := nodeInfo.Node()
 
-		if maxMinDiff == 0 {
+		if minMaxSum == 0 {
```

Comment:
is this still correct? if so what does it mean?

Reply:
It should just be `maxScore`. Fixed.

Reply:
And if maxScore is zero it means we have 0 pods in the cluster, like when we are deploying the first pod.